DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: googleErrorReportingR
-Title: Send Error Reports to the Google Error Reporting Service API
-Version: 0.0.3
+Title: Send error reports to the Google Error Reporting service API
```

Comment:
@andreavargasmon the caps were intentional (at the request of CRAN as feedback on submission). Could you please restore them?

Reply:
done